src/payment/checkout/payment-form/PaymentForm.jsx

```diff
@@ -219,7 +240,12 @@ export class PaymentFormComponent extends React.Component {
           disabled={disabled}
           isPaymentVisualExperiment={isPaymentVisualExperiment}
         />
-        <CardDetails disabled={disabled} isPaymentVisualExperiment={isPaymentVisualExperiment} />
+        <CardDetails
```

Comment:
Looking at how much the code here diverges depending on whether flexMicrofrormEnabled is true or not, it may be worth having two different CardDetails components and move some of the above conditional logic into them.  

When a set of components starts having lots of conditionals like this, it's usually a sign that they're not really one simple component hierarchy, but two separate ones.

I'm not 100% on exactly what that change would look like, but I think it'd make 1) the existing components nearly identical to what they are now, and 2) would put the flex microform code in a component by itself, isolated from the rest of the code that isn't being used in that workflow.